regolith/builders/publistbuilder.py

```diff
@@ -74,8 +75,15 @@ def latex(self):
             cat_grants, all_grants = "", ""
             for g in grants:
                 cat_grants = cat_grants + "_" + g
-            filestub = filestub + "{}".format(cat_grants)
-            qualifiers = qualifiers + " from Grant{} {}".format(pl, text_grants)
+            filestub = f"{filestub}{cat_grants}"
+            qualifiers = f"{qualifiers} from Grant{pl} {text_grants}"
+        if self.rc.kwargs:
+            kw = True
+            key, value = self.rc.kwargs[0].split(':', 1)
+            if key == "facilities":
```

Comment:
this would be a better place for the `facilities = None` to make the code more readable.  Just above the if `key == 'facilities'`